### ISSUE TITLE
Antihotness Apparatus

### ISSUE THREAD

**Issue by friznit** [friznit/Unofficial-BDB-Wiki#71]
System Heat notes shamelessly stolen from Zorg's forum post for reference.  Intend to combine with rough guide to LH2 tank mass vs volume and relative dV for nukes (cos they're vaguely related insofar as LH2 is subject to boiloff, which SH-configured parts help to mitigate)

**Cryo Fuels**
Nuclear engines in particular enjoy very high ISP when using LH2 as fuel.  However, LH2 is a low density fuel compare to LFO.
To get equivalent dV with LH2, you'd need a lot more volume (a bigger fuel tank).  Of course the dry mass of tankage weighs more so you get diminishing returns and the large size can be awkward.
Alternatively, consider that an LH2 stage is much lighter than an equivalent LFO stage, so the lower stages have less mass to lift.

**Boil Off**
LH2 is subject to boiloff: cryogenically cooled fuel dissipates due to heating from external sources (engines, radiation, friction, the sun etc).
Boiloff can be mitigated by insulating the tanks or using radiators.

Current Boiloff Balancing (subject to change)
![image](https://user-images.githubusercontent.com/12999059/179044881-fb9858e5-b475-4aba-a1b6-e07fff9391cd.png)

TL/DR:
1. In VAB "Enable Cooling" in PAW on all tanks that require it (i.e. ones that have fuel like LH2 that boils off).
2.  Spam radiators until the loop temperature stablises in the PAW.

Better approach:
1.  Open SH UI and click on the red temp warning indicator.
2. Check the Net Flux value.  This is essentially how much cooling the tank needs.
3. Add radiators to bring the Net Flux value to 0 (or negative to give yourself a little margin).
4. Heat Loops can be used to create separate groups of parts & radiators.  Useful for ringfencing high operating temp parts such as reactors from low operating temp parts like fuel tanks.

Full guide on the SH wiki
